src/org/labkey/test/tests/ExpTest.java

```diff
@@ -160,29 +159,25 @@ public void testSteps()
         selectQuery("exp", "Data"); // Select the one we want to edit
         waitForElement(Locator.linkWithText("edit metadata"), WAIT_FOR_JAVASCRIPT); //on Ext panel
         clickAndWait(Locator.linkWithText("edit metadata"));
-        waitForElement(Locator.xpath("//span[contains(text(), 'Reset to Default')]"), defaultWaitForPage);
-        int lastFieldIndex = getElementCount(Locator.xpath("//input[starts-with(@name, 'ff_label')]")) - 1;
-        Locator lastField = Locator.xpath("//input[@name='ff_label" + lastFieldIndex + "']");
-        click(lastField);
-        click(Locator.xpath("//span").append(Locator.lkButton("Alias Field")));
-        selectOptionByText(Locator.name("sourceColumn"), "RowId");
-        click(Locator.xpath("//span").append(Locator.lkButton("OK")));
+
+        designerPage = new DomainDesignerPage(getDriver());
+        designerPage.click(Locator.button("Alias Field"));
+        click(Locator.button("OK"));
 
         // Make it a lookup into our custom query
-        editor = PropertiesEditor.PropertiesEditor(getDriver()).withTitleContaining("Metadata Properties").find();
-        int fieldCount = getElementCount(Locator.xpath("//input[contains(@name, 'ff_type')]"));
+        int fieldCount = designerPage.fieldsPanel().fieldNames().size();
         assertTrue(fieldCount > 0);
-        row = editor.selectField(fieldCount-1);
-        row.setType(new FieldDefinition.LookupInfo(null, "exp", "dataCustomQuery"));
+        domainRow = designerPage.fieldsPanel().getField(fieldCount-1);
+        domainRow.setType(FieldDefinition.ColumnType.Lookup).setFromSchema("exp").setFromTargetTable("dataCustomQuery" + " (Integer)");
 
         // Save it
-        clickButton("Save", 0);
-        waitForText(WAIT_FOR_JAVASCRIPT, "Save successful.");
-        clickButton("View Data");
+        designerPage.click(Locator.button("Save"));
```

Comment:
Same as above. I believe the DomainDesignerPage has a clickFinish() for this.

Reply:
I switched to this call from clickFinish() due to expectations of navigation to a new page.

Reply:
Got it. Thanks.